haskell/test/Test/Ligo/BaseDAO/FA2.hs

```diff
@@ -269,71 +269,71 @@ validateTokenOwnerScenario originateFn = do
             , tdAmount = 10
             } ]
         } ]
-      callWith param = withSender (AddressResolved owner1) $
+      callWith param _sender = withSender (AddressResolved _sender) $
         call dao (Call @"Transfer") param
 
-  callWith (params unfrozenTokenId)
+  callWith (params frozenTokenId) admin
 
-  callWith (params frozenTokenId)
+  callWith (params frozenTokenId) owner1
     & expectCustomErrorNoArg #fROZEN_TOKEN_NOT_TRANSFERABLE
-  callWith (params unknownTokenId)
+  callWith (params unknownTokenId) admin
     & expectCustomError_ #fA2_TOKEN_UNDEFINED
 
 
-noForeignMoneyScenario :: MonadNettest caps base m => OriginateFn m -> m ()
-noForeignMoneyScenario originateFn = do
-  ((owner1, op1), (owner2, _), dao, _) <- originateFn
-
-  let params = [ FA2.TransferItem
-        { tiFrom = owner2
-        , tiTxs = [ FA2.TransferDestination
-            { tdTo = owner1
-            , tdTokenId = unfrozenTokenId
-            , tdAmount = 10
-            } ]
-        } ]
-  withSender (AddressResolved op1) $ call dao (Call @"Transfer") params
-    & expectCustomError_ #fA2_NOT_OPERATOR
-
-noForeignMoneyOwnerScenario :: MonadNettest caps base m => OriginateFn m -> m ()
-noForeignMoneyOwnerScenario originateFn = do
-  ((owner1, _), (owner2, _), dao, _) <- originateFn
-
-  let params = [ FA2.TransferItem
-        { tiFrom = owner2
-        , tiTxs = [ FA2.TransferDestination
-            { tdTo = owner1
-            , tdTokenId = unfrozenTokenId
-            , tdAmount = 10
-            } ]
-        } ]
-  withSender (AddressResolved owner1) $ call dao (Call @"Transfer") params
-    & expectCustomError_ #fA2_NOT_OPERATOR
+-- Disabled for now since there is no unfrozen token available
```

Comment:
Please add a `TODO` as in https://github.com/tqtezos/baseDAO/pull/181/files#r608147607

Reply:
Done.